PrebidMobile/PrebidMobile-rendering/src/main/java/org/prebid/mobile/rendering/bidding/display/PrebidMediationDelegate.java

```diff
@@ -0,0 +1,22 @@
+package org.prebid.mobile.rendering.bidding.display;
+
+import androidx.annotation.Nullable;
+import org.prebid.mobile.rendering.bidding.data.bid.BidResponse;
+
+import java.util.HashMap;
+
+public interface PrebidMediationDelegate {
```

Comment:
Need to add detailed comments to the interface and its methods, since it could be used by publishers for supporting custom mediation platforms